src/app/AppContainer.js

```diff
@@ -145,8 +166,8 @@ class AppContainer extends React.Component<PropsType> {
 	}
 
 	componentDidUpdate( prevProps ) {
-		if ( ! prevProps.isReady && ( prevProps.isReady !== this.props.isReady ) ) {
-			const { blocks } = this.props;
+		if ( ! prevProps.isReady && this.props.isReady ) {
```

Comment:
The condition seems to have lost the `prevProps.isReady !== ` part. Is that intentional?

Reply:
Yeah, given that isReady is a boolean, I find it clearer with this syntax